php/src/php/Proxy.cpp

```diff
@@ -1613,14 +1617,24 @@ handleFreeStorage(zend_object* object)
 #ifdef _WIN32
 extern "C"
 #endif
+
 static zend_object*
+#if PHP_VERSION_ID >= 80000
+handleClone(zend_object* zobj)
+#else
 handleClone(zval* zv)
+#endif
 {
     //
     // Create a new object that shares a C++ proxy instance with this object.
     //
 
+#if PHP_VERSION_ID >= 80000
+    ProxyPtr obj = *Wrapper<ProxyPtr>::fetch(zobj)->ptr;
```

Comment:
don't we need to check for null before dereferencing the pointer?

Reply:
This ptr is never null